Syntax checking for JSON and converting to Graph format

borges-filipe opened the pull request.

kristikotini commented on Project/backend/codebase/tests/test_json_to_graphml.py:
```diff
@@ -0,0 +1,89 @@
+    assert graph["C"]["D"]["label"] == "label2"
+
+
+if __name__ == "__main__":
+    pytest.main()
```
You can remove this line since I do not think it is necessary

borges-filipe replied:
Makes sense, removed

kristikotini commented on Project/backend/codebase/graphCreator/json_to_graphml.py:
```diff
@@ -0,0 +1,85 @@
+import json
+import networkx as nx
+import pandas as pd
```
These 2 packages are not included in `requirements.txt`. Please update the `requirements.txt`

borges-filipe replied:
Added them now

kristikotini commented on Project/backend/codebase/tests/test_json_to_graphml.py:
```diff
@@ -0,0 +1,89 @@
+import pytest
+import networkx as nx
+from ..graphCreator import json_to_graphml
```
Maybe the path of import should be different having in mind that the base cwd would be codebase. Try out the tests locally and see if they pass. If the import is not causing problems you can leave it as it is.

borges-filipe replied:
The import path works when running `make test` from the backend folder.

kristikotini reviewed (changes requested):
Hey @borges-filipe , some changes /suggestions below:
- requirements.txt need to be updated 
- some comments for a couple of changes to the tests (run tests locally via command `make test`, for the moment I do not know why the actions are passing when tests are causing errors need to have a look for it in the future)
- I would suggest to rename `graphCreator`-> `graph_creator` (naming convetion in python is snake_case, in `develop` this change is present maybe pull and merge develop and refactor the imports/file structure)

borges-filipe commented:
Hey @kristikotini I just changes the following:

- Updated requirements.txt
- Changed graphCreator to graph_creator
- Removed unnecessary test lines
